ci/docker/ubuntu-20.04-nvidia-cpp.dockerfile

```diff
@@ -1,10 +1,10 @@
 
-ARG cuda=10.1
+ARG cuda=11.4
```

Comment:
It looks like `${cuda}` is never used. Either drop it, or match 11.2 as the default.

Reply:
Removed